actionview/app/assets/javascripts/README.md

```diff
@@ -0,0 +1,49 @@
+Ruby on Rails unobtrusive scripting adapter.
+========================================
+
+This unobtrusive scripting support file is developed for the Ruby on Rails framework, but is not strictly tied to any specific backend. You can drop this into any application to:
+
+- force confirmation dialogs for various actions;
+- make non-GET requests from hyperlinks;
+- make forms or hyperlinks submit data asynchronously with Ajax;
+- have submit buttons become automatically disabled on form submit to prevent double-clicking.
+
+These features are achieved by adding certain ["data" attributes][data] to your HTML markup. In Rails, they are added by the framework's template helpers.
+
+Requirements
+------------
+
+- HTML5 doctype (optional).
+
+If you don't use HTML5, adding "data" attributes to your HTML4 or XHTML pages might make them fail [W3C markup validation][validator]. However, this shouldn't create any issues for web browsers or other user agents.
+
+Installation using npm
+------------
+
+Run `npm install rails-ujs --save` to install the rails-ujs package.
+
+Installation using Yarn
+------------
+
+Run `yarn add rails-ujs` to install the rails-ujs package.
+
+Usage
+------------
+
+Require `rails-ujs` into your application.js manifest.
```

Comment:
@rafaelfranca I was trying to add a CHANGELOG for this PR but I have a doubt.

Since rails/rails now ships with coffeescript files for rails-ujs (like [confirm.coffee](https://github.com/rails/rails/blob/192db64452d148c7b51713979459e38407380dc6/actionview/app/assets/javascripts/features/confirm.coffee)), do users still need to manually install `rails-ujs` through npm or yarn, as indicated above?

Or they only need that if they don't use coffeescript in their Rails app?

Thanks!